geomstats/_backend/pytorch/autodiff.py

```diff
@@ -39,45 +63,76 @@ def backward(ctx, grad_output):
                 inputs = ctx.saved_tensors
 
                 grads = ()
-                for custom_grad in grad_func:
+                for custom_grad in grad_funcs:
                     grads = (*grads, grad_output * custom_grad(*inputs))
 
                 if len(grads) == 1:
                     return grads[0]
                 return grads
 
-        def wrapper(*args, **kwargs):
+        def wrapped_function(*args, **kwargs):
             new_inputs = args + tuple(kwargs.values())
             return func_with_grad.apply(*new_inputs)
 
-        return wrapper
+        return wrapped_function
 
     return decorator
 
 
 def jacobian(func):
-    """Return a function that returns the jacobian of a function."""
+    """Return a function that returns the jacobian of func.
+
+    Parameters
+    ----------
+    func : callable
+        Function whose Jacobian is computed.
+
+    Returns
+    -------
+    _ : callable
+        Function taking x as input and returning
+        the jacobian of func at x.
+    """
     return lambda x: torch_jac(func, x)
 
 
 def value_and_grad(func, to_numpy=False):
-    """Return a function that returns both value and gradient.
+    """Return a function that returns func's value and gradients' values.
 
-    Suitable for use in scipy.optimize
+    Suitable for use in scipy.optimize.
 
     Parameters
     ----------
     func : callable
-        Function to compute the gradient. It must be real-valued.
+        Function whose value and gradient values
+        will be computed.
+    to_numpy : bool
+        Unused. Here for API consistency.
```

Comment:
This is actually used below

Reply:
I actually added that parameter in the docstring in my ongoing PR #1220

Reply:
Done.